client/app/bundles/course/group/pages/GroupShow/GroupManager/GroupUserManager.jsx

```diff
@@ -118,22 +130,65 @@ const getAvailableUsers = (
   groups,
   group,
   hideInGroup,
+  hidePhantomStudent,
   availableSearch,
 ) => {
+  let groupMemberIds = new Set()
```

Comment:
I think a cleaner way to do this is to start with the full list of the group member ids and sequentially remove unfiltered ids. You can try it first and I will improve later.

Reply:
I am using the ternary operation instead. Including it in the commit and maybe later we can see which one actually is better?